components/css/global/_base.scss

```diff
@@ -10,77 +10,66 @@ body{
 	body{
 		font-family:$font-arial;
 		font-size:1em;
-		line-height:24/16*1em;
+		line-height:calc-toEm(24);
 	}
 }
 @media (max-width:767px){
 	body{
-		font-size:18/16*1em;
-		line-height:24/18*1em;
+		font-size:calc-toEm(18);
+		line-height:calc-toEm(24);
 	}
 }
 h1{
 	padding-top:0;
 	padding-bottom:0.25em;
-	font-family:$font-archivo-narrow;
-	font-weight:bold;
+	font-family:$font-champion-welter;
 	color:$color-headlineGold;
-    font-size:36/16 * 1em;
+    font-size: calc-toEm(38);

```

Comment:
Although they don't currently, perhaps the heading font-sizes should change based on the window width. The main heading could there possibly being a problem with larger words in a `h1` bleeding off mobile devices.

Reply:
I'll loop Garrett in and have him mock something up for mobile (and maybe tablet)